### ISSUE TITLE
Need help in Training a Cutom Model

### ISSUE THREAD

**Issue by LasalJayawardena** [aimakerspace/PeekingDuck#628]
I want to build a YOLO model by training on a set of images (training set).  Then save the best model by validation loss (Like Tensorflow Checkpointer).  Finally, I want to use it for predictions. 

How can I implement this? I am a bit unclear about the training phase and using the saved trained model. Any help is appreciated!
Thank you in Advance.

**Comment by leeping-ng** [comment]
Hi! PeekingDuck is designed for model inference rather than model training, so I suggest finding a YOLO repo that comes with a training script to train your model separately. Subsequently, you can create a custom PeekingDuck node for your trained model by referring to [this guide](https://peekingduck.readthedocs.io/en/stable/getting_started/03_custom_nodes.html), and use PeekingDuck for inference. Our [model.yolo_license_plate](https://peekingduck.readthedocs.io/en/stable/peekingduck.pipeline.nodes.model.yolo_license_plate.Node.html#peekingduck.pipeline.nodes.model.yolo_license_plate.Node) node was created using this method.

**Comment by leeping-ng** [comment]
Hi @LasalJayawardena, we have released a [new tutorial](https://peekingduck.readthedocs.io/en/stable/tutorials/06_using_your_own_models.html) on how you can use your own trained models with PeekingDuck. Hope that this helps with your project.
